src/Backend/Enums.js

```diff
@@ -1,6 +1,9 @@
 const ProgramEnum = {
-	MGTE: 'Management Engineering',
-	CHE: 'Chemical Engineering'
+	MGTE:'Management Engineering',
```

Comment:
Please add a space after `:` 
for example: `MGTE: 'Management Engineering'` instead of `MGTE:'Management Engineering'`
Same for others